x-pack/plugins/snapshot_restore/public/app/sections/home/snapshot_list/snapshot_table/snapshot_table.tsx

```diff
@@ -159,6 +166,14 @@ const SnapshotTableUi: React.FunctionComponent<Props> = ({
         })),
       },
     ],
+    defaultQuery: filterToRepository
+      ? Query.parse(`repository:'${filterToRepository}'`, {
```

Comment:
passing a query object with a strict schema is needed here because it kept parsing strings like `repository:'with spaces'` and `repository:'with/slash'` as dates and failed to work

Reply:
Do you think it's worth filing an EUI bug?

Reply:
added comment to existing bug: https://github.com/elastic/eui/issues/1855

Reply:
EuiSearchBar / Query are treat single-quoted values as dates. You want to use double quotes instead (though your forward slash will still not be accepted until https://github.com/elastic/eui/issues/1855 is resolved)